test/tests/functional/pbs_node_rampdown.py

```diff
@@ -630,23 +634,9 @@ def test_release_nodes_on_stageout_true(self):
         # Deleting the job will trigger the stageout process
         # at which time sister nodes are automatically released
         # due to release_nodes_stageout=true set
+        time.sleep(3)
```

Comment:
instead of 3 sec sleep can we check test.img file size? once it 2 gb we can do qdel.

Reply:
Hi @kjakkali We don't know the exact time for creating a 2 gb file. It might take more time to create on slow machines which leads increase in test execution time..

Reply:
@shilpakodli : I think what @kjakkali is suggesting is that instead of doing the sleep 3, just test for existence of  2gb size of test.img file on the stagein host. Perhaps retry it some set number of times, and if unable to see that size, then fail the test...